### PR TITLE
Guide: Scaling for Roomscale WebVR (draft)

### PR DESCRIPTION
**Pull request by 321C4**
Need another set of eyes on this roomscale scaling guide in the "Creating" section --  almost publishable.

### CODE REVIEW

**Comment by cvan** [comment]
@caseyyee @darkwing if either of you get a chance, could you give this a review? thanks!

File: creating/roomscale-scaling/index.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+Blender, A-Frame and Unity all use "1 meter" as their standard unit of measurement. But that does not mean that 1 meter in any of these contexts will map accurately and consistently with real world space, when viewed in VR. If you need accurate real-world scaling of your VR content, it's worth testing the scale with your system.
+
+### Using the demo models
+
+![Demo models: coffee mug and classic banana](/creating/roomscale-scaling/img/demo-viewing.jpg)
+
+In the demo scene, you'll find a few 3D models that can be used for realistic scale references. The meter stick is actually 1 meter long in real space, when viewed in Vive. In A-Frame, it is 1.026 units long. The same is true for units in Blender. Depending on the experience you are building, 2.6cm per meter might not make much difference to your users. However, if you use architectural models in your scene, tables may seem a bit too short, and hallways a bit narrower than we're accustomed to.  
+
+### DIY Measurements
```
Comment (darkwing) [inline creating/roomscale-scaling/index.md]: "DIY" may not translate well for non-native English speakers.  We may want to spell it out.

**Comment by cvan** [comment]
thanks! merging now, will update post changes in `master` 👍

**Comment by caseyyee** [comment]
awesome thanks!

**Pull request merged**